native/src/routes/search/components/NothingFoundFeedbackBox.js

```diff
@@ -65,16 +65,19 @@ const NothingFoundFeedbackBox = ({ resultsFound, query, t, theme, sendFeedback }
   if (['idle', 'failed'].includes(sendingStatus)) {
     return (
       <Container>
-        <FeedbackBox>
-          <TitleText>{resultsFound ? t('feedback:informationNotFound') : t('feedback:nothingFound')}</TitleText>
+        <FeedbackBox theme={theme}>
```

Comment:
Why these changes? I just removed it everywhere :see_no_evil:

Reply:
I added a test, and flow was complaining that the theme could not be found.

Reply:
Just checked it out, I don't get any errors if I just revert this file.

Reply:
sorry my fault, it is not flow. The test is failing when reverting this:

SearchModal › search modal should send tracking signal on Press

    TypeError: Cannot read property 'backgroundAccentColor' of undefined

Reply:
Yes, you need to wrap it in a ThemeProvider, example from web: https://github.com/Integreat/integreat-app/blob/a32091c33c10148a3617d6c873a304877788f96d/web/src/modules/layout/containers/__tests__/LocationHeader.spec.js

Reply:
I think this is currently only possible for web?

https://github.com/Integreat/integreat-app/pull/221#discussion_r600587803

Reply:
Done here: https://github.com/Integreat/integreat-app/pull/227/files#r601429935